pennylane/devices/qutrit_mixed/utils.py

```diff
@@ -0,0 +1,72 @@
+# Copyright 2018-2024 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Functions and variables to be utilized by qutrit mixed state simulator."""
+import functools
+from string import ascii_letters as alphabet
+import pennylane as qml
+from pennylane import numpy as np
+
+alphabet_array = np.array(list(alphabet))
+qudit_dim = 3  # specifies qudit dimension
```

Comment:
Put this variable here to be used throughout

Reply:
```suggestion
QUDIT_DIM = 3  # specifies qudit dimension
```

The convention for global constants in Python is having everything capitalized:

https://peps.python.org/pep-0008/#constants

Reply:
Oh thank you, I forgot